CHANGES.rst

```diff
@@ -1,7 +1,7 @@
 0.12.0 (unreleased)
 ===================
 
-- No changes yet.
+- Dropped support for Python 3.7. [#60]
```

Comment:
Extra level of bikeshedding: Usually we state the minimum supported version rather than what was dropped.

Reply:
Updated. Looks good now? Thanks for review!